catalog/dags/elasticsearch_cluster/shared.py

```diff
@@ -0,0 +1,11 @@
+from airflow.decorators import task
+from airflow.models.connection import Connection
+from airflow.models.xcom_arg import XComArg
+
+from common.constants import Environment
```

Comment:
FYI I am working on a PR where I move this task into a new file called `common/elasticsearch.py` instead.

Reply:
Gotcha! Should I update this PR to do that as well or do you have a different approach you would prefer?

Reply:
It looks like mine will take a minute to get over the line -- I'm totally fine to keep it here for this PR and I can rebase/update in mine.